trusted-types/script-enforcement-004.html

```diff
@@ -79,6 +79,17 @@
     assert_array_equals(messages, ["SET", "APPEND", "CREATE_SCRIPT", "SVGScriptElement text", "RUN_TRUSTED_SCRIPT"]);
   }, "Default policy's calls when setting script source via SVGScriptElement.textContent.");
 
+  promise_test(async t => {
+    let messages = await script_messages_for(_ => {
+      let script = document.createElement("script");
+      window.log_message("SET");
+      Node.prototype.__lookupSetter__("textContent").call(script, LOG_RUN_MESSAGE);
+      window.log_message("APPEND");
+      container.appendChild(script);
+    });
+    assert_array_equals(messages, ["SET", "APPEND", "CREATE_SCRIPT", "HTMLScriptElement text", "RUN_TRUSTED_SCRIPT"]);
+  }, "Default policy's calls when setting script source via Node.prototype.__lookupSetter__('textContent').");
```

Comment:
Slightly hard to understand this description the possessive followed by plural is tripping me up.